tests/NLog.UnitTests/Targets/Wrappers/ConcurrentRequestQueueTests.cs

```diff
@@ -31,6 +31,8 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 // 
 
+#if NET4_5 || NET4_0
```

Comment:
@snakefoot any idea why I suddenly need this one? 

Otherwise i got:

> msbuild.exe tests\NLog.UnitTests /nologo /nr:false /p:DebugType=Full  /p:targetFramework=net452 /p:TestTargetFramework=net35 /verbosity:minimal /p:OutputPath=.\bin\release\net35 /p:platform="Any CPU"
  building net45 ...
  NLog -> D:\nlog\nlog\src\NLog\bin\release\net35\NLog.dll
  NLogAutoLoadExtension -> D:\nlog\nlog\tests\NLogAutoLoadExtension\bin\release\net35\NLogAutoLoadExtension.dll
  SampleExtensions -> D:\nlog\nlog\tests\SampleExtensions\bin\release\net35\SampleExtensions.dll
  PackageLoaderTestAssembly -> D:\nlog\nlog\tests\PackageLoaderTestAssembly\bin\release\net35\PackageLoaderTestAssembly.dll
Targets\Wrappers\ConcurrentRequestQueueTests.cs(52,13): error CS0246: The type or namespace name 'ConcurrentRequestQueue' could not be found (are you missing a using directive or an assembly referenc e?) [D:\nlog\nlog\tests\NLog.UnitTests\NLog.UnitTests.csproj]

Reply:
No I do not. But it makes sense that NET35 does not know about ConcurrentRequestQueue

Reply:
Yes, but strange it was working on AppVeyor.  Maybe there weren't running correctly on AppVeyor?